### PR TITLE
Feature/28 azure pipelines

### PR DESCRIPTION
**Pull request by paulo-teran-jalafund**
[AB#28](https://dev.azure.com/funjaladev/7867a1c5-5ac7-4c7f-baff-87d740bf4fc1/_workitems/edit/28)

### CODE REVIEW

File: azure-pipelines.yml [hunk 1]
```diff
@@ -0,0 +1,44 @@
+# Starter pipeline
+# Start with a minimal pipeline that you can customize to build and deploy your code.
+# Add steps that build, run tests, deploy, and more:
+# https://aka.ms/yaml
+
```
Comment (pedro-lima-jalafund) [inline azure-pipelines.yml]: Could you remove this comments and make `trigger` the first line

File: azure-pipelines.yml [hunk 2]
```diff
@@ -0,0 +1,44 @@
...
+  branches:
+    include:
+      - main
+      - develop
+
+pr:
+  autoCancel: true
+  branches:
+    include:
+      - main
```
Comment (pedro-lima-jalafund) [inline azure-pipelines.yml:10]: Please check the wepApp pipeline file for `draft` on PRs

**Review by pedro-lima-jalafund — COMMENTED** [review]
maybe you could marge the base project to your branch and test with it this pipeline

File: azure-pipelines.yml [hunk 3]
```diff
@@ -0,0 +1,38 @@
...
+  displayName: 'DotNet Install'
+  inputs:
+    packageType: 'sdk'
+    version: '3.x'
+
+- task: DotNetCoreCLI@2
+  displayName: 'Dotnet Build'
+  inputs:
+    command: 'build'
+    projects: '**/**/*.csproj'
```
Comment (martin-callejas-jalafund) [inline azure-pipelines.yml]: Please take out this in a variable, and let's use the `.sln` file as entrypoint

File: test-pipeline/TestCase/TestCase.csproj [hunk 4]
```diff
@@ -0,0 +1,26 @@
+<Project Sdk="Microsoft.NET.Sdk">
```
Comment (martin-callejas-jalafund) [inline test-pipeline/TestCase/TestCase.csproj:1]: Delete all the files below

**Review by martin-callejas-jalafund — CHANGES_REQUESTED** [review]
Nice work! Please work on the observations.

**Pull request closed without merging**